board/main.c

```diff
@@ -577,136 +644,65 @@ int main() {
   // default to silent mode to prevent issues with Ford
   // hardcode a specific safety mode if you want to force the panda to be in a specific mode
   safety_set_mode(SAFETY_NOOUTPUT, 0);
```

Comment:
for VIN query we'll need ELM327 mode, but I guess that can be the default mode set by boardd, while ok for panda to boot in NOOUTPUT.

Reply:
this default is sane